### PR TITLE
sacctmgr: add Chinese translation

### PR DESCRIPTION
**Pull request by zdy023**
* add details w.r.t. the output formats
* add Chinese translation

- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/sacctmgr.md [hunk 1]
```diff
@@ -15,6 +15,6 @@
 
 `sacctmgr add account {{account_name}} cluster={{cluster_of_account}}`
 
-- Show details of user/association/cluster/account:
+- Show details of user/association/cluster/account (in a spcific format):
```
Comment (navarroaxel) [inline pages/linux/sacctmgr.md:18]:
```suggestion
- Show details of user/association/cluster/account using a specific format:
```

File: pages/linux/sacctmgr.md [hunk 2]
```diff
@@ -15,6 +15,6 @@
...
 
-`sacctmgr show {{user/association/cluster/account}}`
+`sacctmgr show {{user/association/cluster/account}} [format="Accout%10" format="GrpTRES%30"]`
```
Comment (navarroaxel) [inline pages/linux/sacctmgr.md:20]:
```suggestion
`sacctmgr show {{user|association|cluster|account}} [format="Accout%10" format="GrpTRES%30"]`
```

File: pages.zh/linux/sacctmgr.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# sacctmgr
+
+> 查看、配置、管理Slurm账户。
```
Comment (blueskyson) [inline pages.zh/linux/sacctmgr.md:3]:
```suggestion
> 查看、配置、管理 Slurm 账户。
```

File: pages.zh/linux/sacctmgr.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+> 更多信息：<https://slurm.schedmd.com/sacctmgr.html>。
+
+- 显示现有配置：
+
+`sacctmgr show configuration`
+
+- 向slurm数据库添加集群：
+
+`sacctmgr add cluster {{cluster_name}}`
```
Comment (blueskyson) [inline pages.zh/linux/sacctmgr.md:12]:
```suggestion
- 向 Slurm 数据库添加集群：

`sacctmgr add cluster {{集群名}}`
```

File: pages.zh/linux/sacctmgr.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- 向slurm数据库添加账户：
+
+`sacctmgr add account {{account_name}} cluster={{cluster_of_account}}`
```
Comment (blueskyson) [inline pages.zh/linux/sacctmgr.md:16]:
```suggestion
- 向 Slurm 数据库添加账户：

`sacctmgr add account {{账户名}} cluster={{账户的集群}}`
```

**Review by blueskyson — COMMENTED** [review]
Please put a space between full-width and half-width characters.

File: pages/linux/sacctmgr.md [hunk 6]
```diff
@@ -15,6 +15,6 @@
 
 `sacctmgr add account {{account_name}} cluster={{cluster_of_account}}`
 
-- Show details of user/association/cluster/account:
+- Show details of user/association/cluster/account using a spcific format:
 
-`sacctmgr show {{user/association/cluster/account}}`
+`sacctmgr show {{user|association|cluster|account}} [format="Accout%10" format="GrpTRES%30"]`
```
Comment (navarroaxel) [inline pages/linux/sacctmgr.md:20]: oh, we don't use `[` to mark optional stuff.
```suggestion
`sacctmgr show {{user|association|cluster|account}} format="Accout%10" format="GrpTRES%30"`
```
Is this a valid example with 2 format arguments?

Comment (zdy023) [reply pages/linux/sacctmgr.md:20]: It is valid. I use such commands to format my output.

File: pages.zh/linux/sacctmgr.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`sacctmgr add cluster {{集群名}}`
+
+- 向 Slurm 数据库添加账户：
+
+`sacctmgr add account {{账户名}} cluster={{账户所在集群}}`
+
+- 以指定格式显示用户、账户资源关联、集群、账户的详细信息：
+
+`sacctmgr show {{user|association|cluster|account}} [format="Accout%10" format="GrpTRES%30"]`
```
Comment (navarroaxel) [inline pages.zh/linux/sacctmgr.md:20]:
```suggestion
`sacctmgr show {{user|association|cluster|account}} format="Accout%10" format="GrpTRES%30"`
```

Comment (zdy023) [reply pages.zh/linux/sacctmgr.md:20]: Sorry, I forgot here.

**Review by blueskyson — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**